docs/source/content/overview/linear_SVM.rst

```diff
@@ -114,13 +130,13 @@ Motivation
 ----------
 
 Why would you ever use SVMs? There are so many different models that can classify data. Why use this one? 
-This is probably the best classifier if you know the data points are easily separable. Also it can be extended
+This is probably the best classifier if you know the data points are easily separable. Also, it can be extended
 by using kernel tricks, so try using the different kernels like Radial Basis Function (RBF). 
 
 
 Code Example
 -------------
-Check out our code, `linear_svm.py`_ to learn how to implement a linear SVM using Python's Scikit-learn library. 
+Check out our code, `here`_ to learn how to implement a linear SVM using Python's Scikit-learn library. 
```

Comment:
Revert this back to linear_svm.py with the link. In my comment I meant the scikit learn link. The `here` should be the link and not the Scikit-Learn